integration_tests/pages/recallSentenceDetails.js

```diff
@@ -35,15 +35,25 @@ const recallSentenceDetailsPage = ({ nomsNumber, recallId } = {}) =>
       cy.get('[id="lastReleasePrison"]').clear().type('Ack')
       cy.contains('Acklington (HMP)').click({ force: true })
     },
+    enterReleasingPrison: nameSubString => {
+      cy.get('[id="lastReleasePrison"]')
+        .clear()
+        .type(nameSubString || 'Ack')
+    },
     setLastReleaseDate: () => {
       cy.get('[name="lastReleaseDateYear"]').clear().type('2020')
       cy.get('[name="lastReleaseDateMonth"]').clear().type('05')
       cy.get('[name="lastReleaseDateDay"]').clear().type('03')
     },
     setSentencingCourt: () => {
-      cy.get('[id="sentencingCourt"]').clear().type('Aberdare')
+      cy.get('[id="sentencingCourt"]').clear().type('Aberd')
       cy.contains('Aberdare County Court').click({ force: true })
     },
+    enterSentencingCourt: nameSubString => {
```

Comment:
this has been repeated in another page object, could move to page.js?

Reply:
Removing: superseded by https://github.com/ministryofjustice/manage-recalls-ui/pull/423#discussion_r759072874